### PR TITLE
Add flake8 GitHub Action to the repo workflow

### PR DESCRIPTION
**Pull request by webknjaz**

### CODE REVIEW

File: .github/main.workflow [hunk 1]
```diff
@@ -0,0 +1,8 @@
+workflow "Code linting" {
+  on = "push"
+  resolves = ["GitHub Action for Flake8"]
+}
+
+action "GitHub Action for Flake8" {
+  uses = "cclauss/GitHub-Action-for-Flake8@0.0.1"
```
Comment (cutwater) [inline .github/main.workflow]: Isn't it required to pass `args` parameter to the workflow file?

Comment (webknjaz) [reply .github/main.workflow]: Depending on whether you want to override the default one. But probably you're right..

Comment (webknjaz) [reply .github/main.workflow]: Done

**Comment by webknjaz** [comment]
@cutwater what are the flake8 settings you use?

**Comment by cutwater** [comment]
@webknjaz we use almost defaults settings, but it seems some extra legacy scripts was checked.
We run `flake8` against `galaxy/` directory.

**Comment by webknjaz** [comment]
@cutwater 
Okay, it should be good to go now!

**Comment by cutwater** [comment]
@webknjaz Looks good. Thanks.

**Pull request merged**